tg_bot.py

```diff
@@ -228,12 +237,16 @@ def start_interval_checking(update, context):
 
     scheduler.add_job(print_available_termins, 'interval', (update, context), minutes=int(minutes), id=user_id)
     scheduled_jobs[user_id] = datetime.datetime.now()
+    scheduled_jobs[limit] = datetime.datetime.now() + datetime.timedelta(days=7)
```

Comment:
No need in this value at all.
First, limit is not global, but specific per user.
Second, subscription start date is already in `scheduled_jobs`, and you can always use it to add 7 days to it

Reply:
Ok, I removed it.